src/site/antora/modules/ROOT/pages/manual/filters.adoc

```diff
@@ -17,1038 +17,1407 @@
+----
+<A onMatch="DENY" onMismatch="NEUTRAL"/>
+----
+
+`A AND B`::
+To select the events that match both `A` and `B` you can use:
++
+[source,xml]
+----
+<Filters>
+  <A/>
+  <B/>
+</Filters>
+----
+
+`A OR B`::
+To select the events that match `A` or `B` we can replace `NEUTRAL` with `ACCEPT` in the `onMatch` attribute:
++
+[source,xml]
+----
+<Filters>
+  <A onMatch="ACCEPT"/>
+  <B onMatch="ACCEPT"/>
+</Filters>
+----
+
+xref:plugin-reference.adoc#org-apache-logging-log4j_log4j-core_org-apache-logging-log4j-core-filter-CompositeFilter[📖 Plugin reference for `Filters`]
 
 [#collection]
 == Collection
 
-Log4j bundles several predefined filters to assist in several common deployment use cases.
-Following sections explain all these in detail.
+Log4j Core provides the following filters out-of-the-box.
+
+[#timestamp-filters]
+=== Timestamp filters
+
+Timestamp filters use the timestamp of log events to decide whether to log them or not.
 
 [#BurstFilter]
-=== BurstFilter
+==== `BurstFilter`
 
-The BurstFilter provides a mechanism to control the rate at which LogEvents are processed by silently discarding events after the maximum limit has been reached.
+The `BurstFilter` uses the _sliding window log_ algorithm to limit the rate of log events.
+The rate limit is only applied to log events less severe than a configured log level.
 
-.Burst Filter Parameters
-[cols="1m,1,4"]
+Besides the common configuration attributes,
+the `BurstFilter` supports the following parameters:
+
+.`BurstFilter` -- configuration attributes
+[cols="1m,1,1,4"]
 |===
-|Parameter Name |Type |Description
-
-|level
-|String
-|Level of messages to be filtered. Anything at or below
-this level will be filtered out if `maxBurst` has been exceeded. The
-default is WARN meaning any messages that are higher than warn will be
-logged regardless of the size of a burst.
-
-|rate
-|float
-|The average number of events per second to allow.
-
-|maxBurst
-|integer
-|The maximum number of events that can occur before
-events are filtered for exceeding the average rate. The default is 10
-times the rate.
-
-|onMatch
-|String
-|Action to take when the filter matches. Can be ACCEPT,
-DENY or NEUTRAL. The default value is NEUTRAL.
-
-|onMismatch
-|String
-|Action to take when the filter does not match. Can
-be ACCEPT, DENY or NEUTRAL. The default value is DENY.
+|Attribute | Type | Default value | Description
+
+| level
+| link:../javadoc/log4j-api/org/apache/logging/log4j/Level.html[`Level`]
+| link:../javadoc/log4j-api/org/apache/logging/log4j/Level.html#WARN[`WARN`]
+| The rate limit only applies to log events less severe than this level.
+Events at least as severe as this level will always match.
+
+| rate
+| `float`
+| `10`
+| The average number of events per second to allow.
+
+| maxBurst
+| `long`
+| `10 &times; rate`
+| The maximum number of events in a sliding window.
+
 |===
 
-A configuration containing the BurstFilter might look like:
+The sliding window can be easily computed as:
```

Comment:
Some users (cf. [this SO question](https://stackoverflow.com/q/72000886/11748454)) want to have more control on the frequency used to log events.

In the case of the `SMTP` appender it really makes sense to have 1 e-mail per hour instead of 24 e-mail at once and then silence.

Fixed in 2cde19bec61964536b33d11635712c9ae8d21fd8.